support-frontend/build.sbt

```diff
@@ -93,6 +93,4 @@ javaOptions in Universal ++= Seq(
 
 javaOptions in Test += "-Dconfig.file=test/selenium/conf/selenium-test.conf"
 
-unmanagedSourceDirectories in Compile += (baseDirectory.value / "assets")
```

Comment:
I'm not sure I understood the full benefits of specifying `assets` as a special directory. I.e. it could be an `unmanagedResourceDirectory` which should also fix this - but I'm unclear on whether that's better than just deleting the line.

Reply:
Is this definitely an issue? Because this was what Flav added to get Intellij to be aware that the assets directory is a source directory I didn't think it had any other implications